guides/source/configuring.md

```diff
@@ -543,6 +543,8 @@ encrypted cookies salt value. Defaults to `'signed encrypted cookie'`.
 
 * `config.action_view.debug_missing_translation` determines whether to wrap the missing translations key in a `<span>` tag or not. This defaults to `true`.
 
+* `config.form_with_generates_remote_forms` determines whether `form_with` generates remote forms or not. This defaults to `true`.
```

Comment:
Should be `config.action_view.form_with_generates_remote_forms`, updated via c2e3e0d2096836bfb6072af70bd0914a960a50fe.